jsapp/js/account/plan.module.scss

```diff
@@ -0,0 +1,232 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+@use 'scss/breakpoints';
+
+.accountPlan {
+  padding: sizes.$x30 sizes.$x40;
+  overflow-y: auto;
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.plansSection {
+  display: grid;
+  grid-template-rows: repeat(4, auto);
+  grid-template-columns: repeat(auto, 1fr);
+  column-gap: sizes.$x20;
+}
+
+.intervalToggle {
+  grid-column: 1 / 5;
+  grid-row: 1/2;
+  justify-self: center;
+  background-color: colors.$kobo-cloud;
+  padding: sizes.$x10 sizes.$x4;
+  border-radius: sizes.$x4;
+}
+
+.intervalToggle input[type="radio"] {
+  display: none;
+  position: absolute;
+  width: 100%;
+  appearance: none;
+}
+
+.intervalToggle label {
+  padding: sizes.$x8 sizes.$x12;
+  color: colors.$kobo-dark-blue;
+}
+
+.intervalToggle input[type="radio"]:checked+label {
+  background-color: colors.$kobo-blue;
+  animation-name: blink;
+  animation-duration: 1s;
+  color: colors.$kobo-white;
+  padding: sizes.$x8 sizes.$x12;
+  border-radius: sizes.$x4;
+}
+
+.intervalToggle input[type="radio"]:checked+label:after {
+  background-color: colors.$kobo-blue;
+  padding: sizes.$x8 sizes.$x12;
+}
+
+.intervalToggle input[type="radio"]:checked+label:before {
+  width: 20px;
+  height: 20px;
+  color: colors.$kobo-cloud;
+  padding: sizes.$x8 sizes.$x12;
+}
+
+.planContainer {
+  grid-auto-columns: 1fr;
+  width: 25em;
+  grid-row: 3/4;
+  border-radius: sizes.$x6;
+  border: colors.$kobo-gray-85 solid 1px;
+  padding: 27px 54px;
+}
+
+.priceName,
+.enterpriseTitle {
+  color: colors.$kobo-gray-40;
+  font-weight: 700;
+  font-size: sizes.$x24;
+}
+
+.priceName {
+  text-align: center;
+}
+
+.enterpriseTitle {
+  margin: 0 0 sizes.$x16;
+
+}
+
+.priceTitle {
+  color: colors.$kobo-dark-blue;
+  text-align: center;
+  font-size: sizes.$x18;
+  padding-bottom: sizes.$x16;
+  font-weight: 600;
+}
+
+.line {
+  border-bottom: 1px solid colors.$kobo-gray-85;
+  border-radius: 5px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+li {
+  padding: sizes.$x6 0;
+  vertical-align: middle;
+  display: flex;
+  font-weight: 400;
+}
+
+.iconContainer {
+  height: 100%;
+  width: auto;
+  display: inline-flex;
+  margin-right: sizes.$x20;
+}
+
+.stormCheck {
+  color: colors.$kobo-storm;
+}
+
+.tealCheck {
+color: colors.$kobo-teal;
+}
+
+.redClose { 
+  color: colors.$kobo-red;
+}
+
+.enterprisePlan {
+  background-color: colors.$kobo-light-blue;
+  grid-column: 1fr;
+  width: 25em;
+  grid-row: 3/4;
+  padding: 39px 46px;
+  height: auto;
+  overflow: hidden;
+  border-radius: sizes.$x6;
+}
+
+.enterpriseDetails {
+  line-height: 24.5px;
+}
+
+.enterpriseLink a {
+  color: colors.$kobo-dark-blue;
+  text-decoration: underline;
+  font-weight: 500;
+}
+
+.listTitle {
```

Comment:
This text should be aligned perfectly with the list items text

Reply:
Also the top margin should be removed and the `<ul>` element should handle margins. I think a clean solution would be to have top margins only on `list + list`. This way the line element would be a nice divider with margins, and then the lists will have gaps between them. Make sure to handle this margin cleanup after you change the list title HTML to be more semantic

Reply:
> 

It should be already, are you seeing something off?

Reply:
> This text should be aligned perfectly with the list items text

It should be already, are you seeing something off?